### PR TITLE
data: AdamsGeeky

### PR DESCRIPTION
**Pull request by AdamsGeeky**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5005"></a>

### CODE REVIEW

**Comment by AdamsGeeky** [comment]
wow Great

File: data/AdamsGeeky.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "ADAMU MUHAMMAD MUHAMMAD",
+  "displayStatsPublic": true,
```
Comment (sansarmaske) [inline data/AdamsGeeky.json:3]:
```suggestion
```
Remove this line as it is no longer needed

File: data/AdamsGeeky.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "type": "personal",
+  "bio": "CS Student **open Source Advocate**. Lead open-source-learners",
+  "avatar": "https://github.com/AdamsGeeky.png"
```
Comment (sansarmaske) [inline data/AdamsGeeky.json:6]:
```suggestion
```
This is also not needed

**Review by sansarmaske — CHANGES_REQUESTED** [review]

File: data/AdamsGeeky.json (hunk 1 above)
Comment (AdamsGeeky) [reply data/AdamsGeeky.json]: thanks

**Pull request merged**